datascience-notebook/Dockerfile

```diff
@@ -43,8 +57,13 @@ RUN conda config --add channels r && \
     'r-nycflights13=0.1*' \
     'r-caret=6.0*' \
     'r-rcurl=1.95*' \
+    'r-dbi=*' \
+    'r-scales=*' \
```

Comment:
Agreed.  Should pin as of versions 4 days ago when I did my last full build and what I'm currently using.  Here are the versions I pulled down:

jovyan@xxx:~/work$ conda list | grep dbi
r-dbi                     0.3.1                 r3.2.2_2a    r
jovyan@xxx:~/work$ conda list | grep scales
r-scales                  0.3.0                 r3.2.2_0a    r